cpu/atmega_common/periph/spi.c

```diff
@@ -61,7 +61,7 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
     DDRB |= ((1 << DDB2) | (1 << DDB1) | (1 << DDB0));
 
     /* make sure the SPI is not powered off */
-    PRR0 &= ~(1 << PRSPI);
+    MEGA_PRR &= ~(1 << PRSPI);
```

Comment:
Since you added `MEGA_PRR` as a macro only for this platform, it will break the other ones using this code. Thus, it is necessary to add the same macro for all the atmega based boards.